### PR TITLE
Workout page

### PR DESCRIPTION
**Pull request by inanimate631**
image
image

### CODE REVIEW

**Review by RetiredHedgehog — APPROVED** [review]

File: backend/authController.js [hunk 1]
```diff
@@ -126,20 +126,25 @@ class authController {
...
+    );
     const objExercise = {
       name: exerciseObj.Name,
       YouTube: exerciseObj['Youtube link'],
       Musclse: exerciseObj['Primary Muscles'],
-      description: description,
+      Type: exerciseObj.Type,
+      Img: `https://img.youtube.com/vi/${videoId}/mqdefault.jpg`,
+      equipment: secondApiResponse.equipment,
+      description: secondApiResponse.instructions,
```
Comment (JaneEllison) [inline backend/authController.js:147]: First, second, ... Not a good names

File: frontend/src/Pages/Workout/components/fill.Workout.template.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  const main = document.querySelector('.menu-workout__list') as HTMLElement;
+  const btn = document.querySelector(`.${sectionName}`) as HTMLElement;
+  btn.classList.add('active__worckout-btn');
+  const trainList = await createTrainList(sectionName);
+  main.innerHTML = trainList;
+}
+
+export async function createTrainList(sectionName: 'Workout' | 'Exercises') {
+  let template = '';
+  const trainList =
```
Comment (JaneEllison) [inline frontend/src/Pages/Workout/components/fill.Workout.template.ts:21]: Train is a 🚂

File: frontend/src/Pages/Workout/styles/Workout.sass [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+.menu-workout__item_link
+  height: 100%
+  display: block
+
+.main__menu-workout
+  margin-top: 66px
+
+.plus
+  top: 40.1%
```
Comment (JaneEllison) [inline frontend/src/Pages/Workout/styles/Workout.sass:48]: Why 40.1?))

File: frontend/src/Pages/Workout/styles/Workout.sass [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    background: $color-black
+    position: absolute
+    left: -22px
+    top: 22px
+
+.menu-workout-title
+  margin-top: 20px
+  margin-left: 24px
+  font-weight: 400
+  font-size: 16px
```
Comment (JaneEllison) [inline frontend/src/Pages/Workout/styles/Workout.sass:68]: Variable

File: frontend/src/Routes/module.ts [hunk 5]
```diff
@@ -19,9 +19,10 @@ export class Module {
   }
 
   renderPage() {
-    const url: string = router.getUrl();
+    const startUrl: string = router.getUrl();
+    console.log(startUrl);
```
Comment (JaneEllison) [inline frontend/src/Routes/module.ts:23]: Do we need this console log?

**Review by JaneEllison — APPROVED** [review]

File: frontend/src/Pages/Workout/styles/Workout.sass (hunk 3 above)
Comment (inanimate631) [reply frontend/src/Pages/Workout/styles/Workout.sass:48]: my bad top = 39.7 because distance to plus = 89px height = 224px and 89*100 / 224 = 39.7 ≈ 40%

**Review by Doroshkeviches — APPROVED** [review]

**Pull request merged**